web-app/src/app/timestamp/timestamp.component.css

```diff
@@ -0,0 +1,57 @@
+.packet {
+  display: flex;
+  width: 90%;
+  flex-direction: row;
+  justify-content: flex-start;
+  background-color: transparent;
+}
+
+.packet-header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 10px;
+}
+.disconnected {
+  color: red;
+
+}
+
+@media screen
+  and (min-width: 769px),
+  screen
+  and (min-height: 600px){
+    :host {
+      display: none;
```

Comment:
This just makes the component not be displayed, so all your timestamp components will disappear if the screen size is width > 769 & h > 600. That's probably why you need to use `display: block` in other places to make it show up.